packages/suite/src/config/suite/features.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * Feature flags configuration file
+ * @docs docs/misc/feature_flags.md
+ */
+
+// List of all feature flags and their explanation
+export type FeatureFlags =
```

Comment:
this type can be generated directly from FLAGS object if you freeze it as const:
`type FeatureFlags = keyof typeof FLAGS;`

so you don't have to write them twice, in type and in object